### DISCUSSION TITLE
如何自定义同文风主题下 liquid keyboard 底部按钮的名字?

### DISCUSSION THREAD

**Discussion by UlyssesZh** [osfans/android-rime#1471] [Q&A | 问答] [answered]
我这边试出来*空格*, *退格*, *剪贴板*这几个按键的名字是能改的, 但是*返回*, *回车*, *更多*好像改不了. 尤其是*回车*, 我在 tongwenfeng.trime.yaml 里面搜索了一下都没搜索到这两个字 (除了注释).

---

编辑: 找到了*返回*是 `preset_keys/liquid_keyboard_exit/label`, *回车*是 `preset_keys/Return2/label`. 但是*更多*仍然没找到. 我尝试了 `preset_keys/liquid_keyboard_switch/label`, 但是没有作用.

**Comment by Maimore** [comment, marked as answer]
![Screenshot_2025-09-10-08-48-33-724_bin mt plus canary-edit](https://github.com/user-attachments/assets/5a858e50-036d-4bb8-87d8-3e0eab9a4c45)
![Screenshot_2025-09-10-08-49-22-005_bin mt plus canary-edit](https://github.com/user-attachments/assets/20395f97-25ed-4d0b-abf6-174ce9563e7a)
